.github/workflows/ci.yml

```diff
@@ -59,17 +59,14 @@ jobs:
           restore-keys: |
             debug-${{ runner.os }}-${{ hashFiles('rust-toolchain') }}-
             debug-${{ runner.os }}-
-            debug-${{ runner.os }}
+            debug-
       - name: Ensure Disk Quota
```

Comment:
Why only ensuring disk quota in style check?

Reply:
Cache is shared among jobs, so we only need to check disk in one job.